### ISSUE TITLE
oxfmt: Support Tailwind css class sorting for @apply in css files

### ISSUE THREAD

**Issue by choi-me** [oxc-project/oxc#19801]
I use the `@apply` keyword in CSS files with Tailwind CSS.
I would like the sorting feature to work within CSS files as well.

Reference test file from prettier-plugin-tailwindcss:
https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/main/tests/tests.ts

**Comment by leaysgur** [comment]
Just to confirm, are you saying that:

- for `.css` files,
- the combination of Prettier and `prettier-plugin-tailwindcss` works,
- but Oxfmt + `sortTailwindcss` option does not

?

**Comment by choi-me** [comment]
> Just to confirm, are you saying that:
> 
> * for `.css` files,
> * the combination of Prettier and `prettier-plugin-tailwindcss` works,
> * but Oxfmt + `sortTailwindcss` option does not
> 
> ?

Exactly. I’m using Oxfmt in VSCode and confirmed that Tailwind class sorting doesn't trigger inside css files.

I submitted this as a feature request, but please let me know if this is actually a bug and it's supposed to be working already.

**Comment by leaysgur** [comment]
Thanks for confirmation.

Could you help us with a bit more information: Is this a VSCode-specific issue, or does it also occur in the CLI?

**Comment by choi-me** [comment]
> Thanks for confirmation.
> 
> Could you help us with a bit more information: Is this a VSCode-specific issue, or does it also occur in the CLI?

It also occurs in the CLI.
I tested it with the latest version of Oxfmt, and the Tailwind classes within the `@apply` directive are not being sorted in css files.

```tsx
// test1.tsx

// Before
export function Comp1() {
  return <div className="left-0 bg-amber-50     absolute"></div>
}

// After CLI (Sorted)
export function Comp1() {
	return <div className="absolute left-0 bg-amber-50"></div>
}
```

But in css file
```css
/* test2.css */
/* Both before and after CLI are same, no change... */

.test1 {
	@apply left-0          absolute top-0;
}
```

**Comment by leaysgur** [comment]
That's a big help! 🙏🏻 

In that case, it's likely a bug... Since `.css` is entirely handled by Prettier and its plugins, we assume it should work.

I will investigate it later.

**Comment by leaysgur** [comment]
Fixed by #19803 , will be released as v0.36. 👍🏻
